plutus-conformance/test-cases/uplc/evaluation/untyped-plutus-core-tests/polyError/polyError.uplc

```diff
@@ -0,0 +1 @@
+(program 1.0.0 (delay (error )))
```

Comment:
Probably also uninteresting. Or I guess it's interesting for a different reason: it shows that delay can block evaluation errors. So maybe just needs a rename.